src/ts/components/Match.tsx

```diff
@@ -8,13 +8,19 @@ interface IMatchProps<TMatch extends IMatch> {
   match: TMatch;
 }
 
+const getFeedbackLink = (feedbackInfo: any) => {
+  const data = encodeURIComponent(JSON.stringify(feedbackInfo, undefined, 2))
+  return "https://docs.google.com/forms/d/e/1FAIpQLSfMOgvJtCchnW0_2zB7Afz_WtYJ5lnPqQI-dgFZ-p0B4h6uKw/viewform?usp=pp_url&entry.110962249=" + data
```

Comment:
Could the form URL be provided as a config value?

Reply:
Just doing that :)